odl/space/pspace.py

```diff
@@ -873,24 +873,42 @@ def show(self, title=None, indices=None, **kwargs):
             else:
                 indices = list(np.linspace(0, self.size - 1, 4, dtype=int))
         else:
-            if isinstance(indices, tuple):
+            if (isinstance(indices, tuple) or
+                    (isinstance(indices, list) and
+                     not all(isinstance(idx, Integral) for idx in indices))):
+                # Tuples or lists containing non-integers index by axis.
```

Comment:
This disagrees with classical np indexing wherein tuples and lists are strictly different in indexing. Do we really want to differ?